dashboard/config/routes.rb

```diff
@@ -280,26 +286,26 @@
 
     get 'instructions', to: 'scripts#instructions'
 
-    # /s/xxx/stage/yyy/puzzle/zzz
-    resources :stages, only: [], path: "/stage", param: 'position', format: false do
+    # /s/xxx/lesson/yyy/puzzle/zzz
+    resources :stages, only: [], path: "/lesson", param: 'position', format: false do
       get 'extras', to: 'script_levels#stage_extras', format: false
       get 'summary_for_lesson_plans', to: 'script_levels#summary_for_lesson_plans', format: false
       resources :script_levels, only: [:show], path: "/puzzle", format: false do
         member do
-          # /s/xxx/stage/yyy/puzzle/zzz/page/ppp
+          # /s/xxx/lesson/yyy/puzzle/zzz/page/ppp
           get 'page/:puzzle_page', to: 'script_levels#show', as: 'puzzle_page', format: false
-          # /s/xxx/stage/yyy/puzzle/zzz/sublevel/sss
+          # /s/xxx/lesson/yyy/puzzle/zzz/sublevel/sss
           get 'sublevel/:sublevel_position', to: 'script_levels#show', as: 'sublevel', format: false
         end
       end
     end
 
-    # /s/xxx/lockable/yyy/puzzle/zzz
-    resources :lockable_stages, only: [], path: "/lockable", param: 'position', format: false do
+    # /s/xxx/survey/yyy/puzzle/zzz
+    resources :lockable_stages, only: [], path: "/survey", param: 'position', format: false do
       get 'summary_for_lesson_plans', to: 'script_levels#summary_for_lesson_plans', format: false
       resources :script_levels, only: [:show], path: "/puzzle", format: false do
         member do
-          # /s/xxx/stage/yyy/puzzle/zzz/page/ppp
+          # /s/xxx/survey/yyy/puzzle/zzz/page/ppp
```

Comment:
This is the other core part of this PR. This updates the script_level paths to use the new URLs going forward